OtripleS.Portal.Web/Brokers/API/ApiBroker.Users.cs

```diff
@@ -0,0 +1,19 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OtripleS.Portal.Web.Models.Users;
+
+namespace OtripleS.Portal.Web.Brokers.API
+{
+    public partial class ApiBroker
```

Comment:
Why is the class partial?
Same for the interface too.

Reply:
Broker functionalities are implemented in partial classes in the project. I have followed the project structure

Reply:
Agreed, the partial keyword is used when you want more than one file for the same class. In this case, we do not have that need. So you can remove it.

Reply:
So, instead of ApiBroker.Users.cs partial class, I need to rename this as UsersApiBroker class. right?

Reply:
I am really sorry for the confusion. Missed the project context. You are good here.